sdk/monitor/monitor-query/swagger/log_query_swagger.json

```diff
@@ -0,0 +1,1438 @@
+{
```

Comment:
feels a little strange having to check this in here

Reply:
Agreed. This is temporary - moving this stuff into the azure-rest repo is in process.

Reply:
Moved (referencing azure-rest-specs-api but using @srnagar's branch)